hardware/raspberrypi/booteeprom.md

```diff
@@ -44,18 +44,38 @@ We will soon be releasing a tool which allows the EEPROM config to be extracted
 
 If 1 then enable UART debug output on GPIO 14 and 15. Configure the debug terminal at 115200bps, 8 bits, no parity bits, 1 stop bit. 
 Default: 0  
-Version: All
+Version: All  
 
 #### WAKE_ON_GPIO 
 
 If 1 then 'sudo halt' will run in a lower power mode until either GPIO3 or GLOBAL_EN are shorted to ground.  
 
-Default: 0 in original version of bootloader (2019-05-10). Newer bootloaders have this set to 1.
+Default: 0 in original version of bootloader (2019-05-10). Newer bootloaders have this set to 1.  
 Version: All  
 
 #### POWER_OFF_ON_HALT  
 
-If 1 and WAKE_ON_GPIO=0 then switch off all PMIC outputs in halt. This is lowest possible power state for halt but may cause problems with some HATS because 5V will still be on. GLOBAL_EN must be shorted to ground to boot.  
+If 1 and WAKE_ON_GPIO=0 then switch off all PMIC outputs in halt. This is lowest possible power state for halt but may cause problems with some HATs because 5V will still be on. GLOBAL_EN must be shorted to ground to boot.  
 
 Default: 0  
 Version: 2019-07-15  
+
+#### FREEZE_VERSION
+
+If 1 then the EEPROM update service (rpi-eeprom package) will skip automatic updates on this board. The parameter is not processed by the bootloader since there is no way in software of fully write protecting the EEPROM. Custom EEPROM update scripts must also check for this flag.
```

Comment:
Does "The parameter is not processed by the bootloader" mean that `recovery.bin` doesn't check the value of this flag?

Reply:
Correct recovery.bin does not look at this, otherwise it would be be (almost) impossible to upgrade the bootloader. It's only the host-script scripts which look at this.

Reply:
Perhaps the text above could be clarified (as it's not obvious what "the bootloader" means in this context)? Something like "This parameter is ignored by `recovery.bin`" ?

Reply:
Done